### ISSUE TITLE
Docs?

### ISSUE THREAD

**Issue by bates64** [brandly/wrenpm#1]
This looks awesome.

**Comment by bates64** [comment]
aaaa nevermind, this repo looks dead.

**Comment by EvanHahn** [comment]
@heyitsmeuralex It is indeed—is this something you're interested in using?

**Comment by bates64** [comment]
Yes! Using and/or contributing - having a definitive solution for managing wren packages would be v. nice - throwing git submodules around and global packages isn't the most fun thing in the world.

We'd need https://github.com/munificent/wren/pull/479 for package management to be viable at all, however - primarily, _relative imports_.

Unless we wrapped the default wren cli, that is.

**Comment by brandly** [comment]
@heyitsmeuralex i sent you an invite to join this repo, if you'd like!

everything that's here so far was written quickly by @EvanHahn and i, before realizing that Wren wasn't quite ready yet. since it looks like there's progress being made on https://github.com/munificent/wren/pull/479, we can revive this soon.

in terms of docs, the closest thing we have is [this TOML file](https://github.com/brandly/wrenpm/blob/master/test/test-package.toml) defining a package. our primary design goal, if i remember correctly, was to just rely on github as our database of sorts and keep the implementation as minimal as possible. there are [questions that haven't been answered](https://github.com/brandly/wrenpm/blob/717fe362450add1bcd41872815c8226a280596c4/lib/install-dependencies.js#L46-L48) yet, so things are pretty loosely defined at this point.

**Comment by bates64** [comment]
Thanks!

Is it worth using something other than Node for this? It'd be nice if people didn't have to have Node.js installed (not everyone does!!) in order to use wrenpm. This codebase is 3 years old, so I'd imagine you two aren't especially happy with its quality at the moment anyway.

---

I just got parsing and displaying `test-package.toml` working in Rust:

![image](https://user-images.githubusercontent.com/9429556/38162938-dfe844f6-34e2-11e8-9236-679303136daf.png)

No idea if this is something worth pursuing, but it looks like [git bindings for Rust](https://docs.rs/git2/0.7.1/git2/) are stronger than Node's, and we get speed and compiled code for free.

Pushed to the `rs` branch if you're interested.
